src/cli/project.py

```diff
@@ -232,55 +291,41 @@ def up(ctx, project, organization, ingress, provider, workers, **kwargs):
     Start/Resume a project cluster.
     """
 
-    # GraphQL
     try:
         graph_ql = GraphQL(authentication=ctx.auth)
-        data = graph_ql.query(
-            """
-            {
-                allProjects {
-                    results {
-                        id
-                        title
-                        clusterSettings {
+        if not organization:
+            data = graph_ql.query(
+                """
+                {
+                    allProjects {
+                        results {
                             id
-                            port
+                            title
+                            clusterSettings {
+                                id
+                                port
+                            }
                         }
                     }
                 }
-            }
-            """
-        )
+                """
+            )
+        else:
+            organization_id = get_organization_id_by_title(graph_ql, organization)
+            if organization_id:
+                data = get_projects_by_organization_id(graph_ql, organization_id)
+            else:
+                console.error("Wrong organization title. Such organization does not exist.")
```

Comment:
`console.error("Wrong organization title. Such organization does not exist.", _exit=True)`